### PR TITLE
Added June 8th meeting minutes

### PR DESCRIPTION
**Pull request by nebrius**
I also fixed a rename of an older meeting. This was fixed earlier, but I think it was reverted accidentally.

### CODE REVIEW

File: meetings/2017-06-08.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* _Previous Minutes Google Doc_:
+https://docs.google.com/document/d/1J14T5lXuFWsaTqfkpmPruAZ4E-WsVe2V5TAaKwDjHN0/
+
+## Present
+* Bryan Hughes @nebrius (CommComm)
+* Tracy Hinds @hackygolucky (CommComm)
+* Tierney Cyren @bnb (observer)
+* Richard Littauer @RichardLitt (observer)
+* Myles Borins @MylesBorins (CommComm)
+* Micheal Dawson  @mhdawson(observer, TSC)
```
Comment (pup) [inline meetings/2017-06-08.md]: missing space before the brackets here!

File: meetings/2017-06-08.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Node.js Foundation Community Committee Meeting 2017-06-08
+
+## Links
+
+* **GitHub Issue**: [community-committee#62](https://github.com/nodejs/community-committee/issues/62)
```
Comment (bnb) [inline meetings/2017-06-08.md]: Would it be possible to start including the video recording link in the Links section here?

File: meetings/2017-06-08.md (hunk 1 above)
Comment (mhdawson) [inline meetings/2017-06-08.md]: as a nit, also an extra space before the @mhdawson

Comment (nebrius) [inline meetings/2017-06-08.md]: Fixed

**Pull request merged**